MapleServer2/PacketHandlers/Game/GuildHandler.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System.Collections.Generic;
+using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Packets;
+using MapleServer2.Servers.Game;
+using Microsoft.Extensions.Logging;
+using MapleServer2.Types;
+
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class GuildHandler : GamePacketHandler
+    {
+        public override RecvOp OpCode => RecvOp.GUILD;
+
+        public GuildHandler(ILogger<GuildHandler> logger) : base(logger) { }
+
+        private enum GuildMode : byte
+        {
+            Create = 0x1,
+            CheckIn = 0xF,
+            EnterHouse = 0x64,
+            GuildWindow = 0x54,
+            List = 0x55,
+            GuildNotice = 0x3E,
+            GuildDonate = 0x6E,
```

Comment:
Can these be in order, in addition to the switch below, and the order of the methods in general